iceoryx2-ffi/ffi/src/api/unique_notifier_id.rs

```diff
@@ -86,6 +86,24 @@ impl HandleToType for iox2_unique_notifier_id_h_ref {
 
 // BEGIN C API
 
+#[no_mangle]
+unsafe extern "C" fn iox2_unique_notifier_id_value(
+    handle: iox2_unique_notifier_id_h,
+    id_ptr: *mut u8,
```

Comment:
I think C best practice is to provide here the length as well. What happens when the user provides insufficient memory - then it runs into a segfault and no one knows why.

But when the length is provided as well we could return an error enum stating - insufficient memory provided!

This applies to all `unique_***_id_value()` functions!